uint8 id
uint32 systemTime
float32 distance
uint8 distanceStatus
uint16 signalStrength
